dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -239,6 +254,53 @@ def test_correct_duplicate_row_count_test(self):
         self.assertEqual(2999, self.trained_schema.total_samples)
         self.assertEqual(0.0, self.trained_schema._get_duplicate_row_count())
 
+    def test_correlation(self):
```

Comment:
should mock out labeler if it isn't auto mocked out.

Reply:
done